botocore/loaders.py

```diff
@@ -319,7 +331,8 @@ def list_api_versions(self, service_name, type_name):
         return sorted(known_api_versions)
 
     @instance_cache
-    def load_service_model(self, service_name, type_name, api_version=None):
+    def load_service_model(self, service_name, type_name, api_version=None,
+                           load_extras=True):
```

Comment:
I don't think we need this `load_extras` argument added here.  This is something we can't remove in the future so unless we actually need this now, I'd prefer to just have a toggle at the `__init__` that allow you to turn this behavior on/off.